### ISSUE TITLE
Not being maintained? I want to take care of it.

### ISSUE THREAD

**Issue by thisishoon** [xiph/rnnoise#106]
There are tons of PRs and a lot of issues are unanswered.

I think rnnoise is great, but it needs more maintainers and activities to fulfill features implementing these days techs.

What is the current status and future plans for rnnoise? @GregorR @jmvalin

**Comment by tdaede** [comment]
Help would be great! Reviewing some PRs would be a good first step.

(I maintain Xiph services)

**Comment by thisishoon** [comment]
You mean just leave the comments in PRs?

**Comment by tdaede** [comment]
Yes. I'll also review them and land them. (I can't add you as an admin without some contribution history)

**Comment by winstondu** [comment]
@thisishoon , one thing that would be helpful is a well-trained model! The current model isn't great at processing voice when the noise are sharp taps.

**Comment by mysteryjeans** [comment]
Where is the model file, am I missing something?
